eta/core/video.py

```diff
@@ -1382,6 +1382,30 @@ def _apply_schema(self):
         for video_labels in self:
             self._apply_schema_to_video(video_labels)
 
+    @classmethod
+    def from_video_labels_patt(cls, video_labels_patt):
+        '''Creates an instance of VideoSetLabels from a pattern of VideoLabels
+        files.
+
+        Args:
+             video_labels_patt: a pattern with one or more numeric sequences:
+                example: "/path/to/labels/%05d.json"
+
+        Returns:
+            a VideoSetLabels instance
+
+        '''
+        labels_list = [
+            VideoLabels.from_json(labels_path)
+            for labels_path in etau.get_pattern_matches(video_labels_patt)
+        ]
+
+        video_set_labels = VideoSetLabels()
```

Comment:
Can you use `cls()` here? For `BigVideoSetLabel`'s sake. My comment applies to `from_image_labels_patt()`, as well.